src/IECoreHoudini/bindings/IECoreHoudiniModule.cpp

```diff
@@ -187,7 +189,10 @@ BOOST_PYTHON_MODULE(_IECoreHoudini)
 
 	IECorePython::PointerFromSWIG<HOM_Geometry>();
 
-#if UT_MAJOR_VERSION_INT >= 14
+// in QT5, it's not possible to share the GL context in the same way.
+#if MIN_HOU_VERSION(16, 5, 0)
```

Comment:
Have we confirmed that with SideFx? @johnhaddon @donboie any thoughts on how we'd get Caribou for Houdini working without this?

Reply:
At the moment, IECoreGL assumes that any resources it creates will be useable in any of the GL contexts it is used in. That means that all of those contexts must share the same resource context, which is what this code is making possible, in conjunction with [code in Gaffer](https://github.com/GafferHQ/gaffer/blob/master/python/GafferUI/GLWidget.py#L305). Without this ability to do sharing, we would need to do one of the following :

1) Not use IECoreGL in any Houdini-owned GL contexts (so no GR_CortexPrimitive).
2) Allow IECoreGL to manage different pools of resources, and use a different pool in Houdini vs in Gaffer.